src/configs/dart.rs

```diff
@@ -13,7 +13,7 @@ pub struct DartConfig<'a> {
 impl<'a> RootModuleConfig<'a> for DartConfig<'a> {
     fn new() -> Self {
         DartConfig {
-            format: "via [$symbol$version]($style) ",
+            format: "via [$symbol$version ]($style)",
```

Comment:
You need to wrap this in braces so when `$version` is missing the extra space doesn't get printed.

```suggestion
            format: "via [$symbol($version )]($style)",
```